### PR TITLE
test(compute): add group resize test, remove int64

### PR DESCRIPTION
**Pull request by georgiyekkert**
1) add a test for https://b.corp.google.com/issues/208483578
2) remove int64 field test - GCP allows only 6 images per hour, so the test fails if we run it more than 6 times.

### CODE REVIEW

File: compute/apiv1/smoke_test.go [hunk 1]
```diff
@@ -630,3 +568,184 @@ func TestHeaders(t *testing.T) {
...
+		&computepb.GetInstanceGroupManagerRequest{
+			Project:              projectId,
+			Zone:                 defaultZone,
+			InstanceGroupManager: managerName,
+		})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if fetched.GetTargetSize() != 0 {
+		t.Fatal(fmt.Sprintf("expected target size: %d, got: %d", 0, fetched.GetTargetSize()))
```
Comment (noahdietz) [inline compute/apiv1/smoke_test.go:749]:
```suggestion
		t.Fatalf("expected target size: %d, got: %d", 0, fetched.GetTargetSize())
```

File: compute/apiv1/smoke_test.go [hunk 2]
```diff
@@ -630,3 +568,184 @@ func TestHeaders(t *testing.T) {
...
+		&computepb.GetInstanceGroupManagerRequest{
+			Project:              projectId,
+			Zone:                 defaultZone,
+			InstanceGroupManager: managerName,
+		})
+	if err != nil {
+		t.Fatal(err)
+	}
+	if fetched.GetTargetSize() != 1 {
+		t.Fatal(fmt.Sprintf("expected target size: %d, got: %d", 1, fetched.GetTargetSize()))
```
Comment (noahdietz) [inline compute/apiv1/smoke_test.go:715]:
```suggestion
		t.Fatalf("expected target size: %d, got: %d", 1, fetched.GetTargetSize())
```

**Review by codyoss — APPROVED** [review]
LGTM

**Comment by georgiyekkert** [comment]
@codyoss could you merge it please?

**Pull request merged**